src/NuGet.Core/NuGet.Build.Tasks/NuGet.props

```diff
@@ -38,4 +38,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     <CentralPackageVersionsFileImported>true</CentralPackageVersionsFileImported>
   </PropertyGroup>
 
+  <!-- Extension point after Directory.Packages.props is imported. -->
```

Comment:
Do we need a _before_ `Directory.Packages.props` extension point as well?  Or is `Directory.Build.props` good enough?

Reply:
For our use case we only need an extension point after Directory.Packages.props is imported but I'm fine with adding a before one as well.

Reply:
I think this is good enough